src/__tests__/Button.test.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Button from '../components/Button/Button';
+
+it('renders without crashing', () => {
+  const div = document.createElement('div');
+  ReactDOM.render(<Button value="Submit" />, div);
```

Comment:
This test has no assertions. Could you assert the generated markup matches what you expect it to return.